pkg/router/controller/status.go

```diff
@@ -45,8 +45,14 @@ func NewStatusAdmitter(plugin router.Plugin, client client.RoutesNamespacer, nam
 	}
 }
 
+// Return a time truncated to the second to ensure that in-memory and
+// serialized timestamps can be safely compared.
+func getRFC3339Timestamp() unversioned.Time {
+	return unversioned.Time{Time: unversioned.Now().Truncate(time.Second)}
```

Comment:
There's an api call available called `Rfc3339Copy` which returns time at the second level precision  - that should work here, no? 
Basically:  `return unversioned.Now().Rfc3339Copy()`
Ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/unversioned/time.go#L76